cq-prolyfill.js

```diff
@@ -317,10 +318,26 @@ function preprocessSheet(sheet, callback) {
  *                                    success or empty string on failure
  */
 function loadExternal(href, callback) {
+	var cacheEntryType = typeof requestCache[href];
+	if (cacheEntryType === 'string') {
+		callback(requestCache[href]);
+		return;
+	}
+	else if (cacheEntryType === 'object') {
```

Comment:
`requestCache[href]` will be `undefined` initially, an array until the request is finished and a string afterwards. Failed XHRs are saved as an empty string, so it should never be `null`.